tests/integration/galley/statusupdate/statusupdate_test.go

```diff
@@ -0,0 +1,110 @@
+package statusupdate
+
+import (
+	"fmt"
+	"testing"
+
+	. "github.com/onsi/gomega"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+
+	"istio.io/istio/galley/pkg/config/analysis/msg"
+	"istio.io/istio/pkg/test/framework"
+	"istio.io/istio/pkg/test/framework/components/environment"
+	"istio.io/istio/pkg/test/framework/components/environment/kube"
+	"istio.io/istio/pkg/test/framework/components/istio"
+	"istio.io/istio/pkg/test/framework/label"
+)
+
+func TestMain(m *testing.M) {
+	framework.
+		NewSuite("statusupdate_test", m).
+		Label(label.CustomSetup).
+		SetupOnEnv(environment.Kube, istio.Setup(nil, setupConfig)).
+		Run()
+}
+
+func setupConfig(cfg *istio.Config) {
+	if cfg == nil {
+		return
+	}
+	cfg.Values["galley.enableAnalysis"] = "true"
+}
+
+const (
+	serviceRoleBindingYaml = `
+apiVersion: rbac.istio.io/v1alpha1
+kind: ServiceRoleBinding
+metadata:
+  name: service-role-binding
+  namespace: default
```

Comment:
Please don't use namespace default. Instead, create a new namespace programmatically and use that instead. There are test framework apis for this.

Reply:
Done